akka-javasdk/src/main/java/akka/javasdk/http/RequestBuilder.java

```diff
@@ -36,6 +37,10 @@ public interface RequestBuilder<R> {
 
   RequestBuilder<R> withTimeout(Duration timeout);
 
+  RequestBuilder<R> withQuery(String key, String value);
+
+  RequestBuilder<R>  withQuery(Query query);
```

Comment:
I guess you added this to be able to add multiple parameters, maybe skip this in the public API for adding the Akka HTTP query type and have a `addQueryParameter` instead?

Reply:
Not sure if I get what you mean. What do you suggest `addQueryParameter` to look like? Wouldn't it be enough if we remove this method and rename the `withQuery(String, String)` into addQueryParameter?

Reply:
`withNnn` is expected to replace (and impl does) so subsequent calls drops the previous query. `addNnnn` is expected to add to existing so would allow adding multiple query parameters in a sequence: 

```
httpClient.POST("/customer/" + id)
  .addQueryParameter("one", "val1")
  .addQueryParameter("two", "val2")
  .invoceAsync()
```